Classes/Controller/ImageLinkRenderingController.php

```diff
@@ -82,26 +82,36 @@ public function renderImages($conf)
             // Get image attributes
             preg_match_all($attrSearchPattern, $passedImage, $passedAttributes);
             $passedAttributes = array_combine($passedAttributes[1], $passedAttributes[2]);
-            // Remove empty values
-            $passedAttributes = array_filter($passedAttributes);
 
             if (!empty($passedAttributes['data-htmlarea-file-uid'])) {
                 try {
                     $systemImage = Resource\ResourceFactory::getInstance()->getFileObject($passedAttributes['data-htmlarea-file-uid']);
+
                     $imageConfiguration = [
                         'width' => ($passedAttributes['width']) ? $passedAttributes['width'] : $systemImage->getProperty('width'),
                         'height' => ($passedAttributes['height']) ? $passedAttributes['height'] : $systemImage->getProperty('height')
                     ];
+
+                    // Title/Alttext: when override mode is enabled use the fields value even if it's empty
+                    // When value is empty, attribute will be removed
+                    $title = ($passedAttributes['title'] || (array_key_exists('title', $passedAttributes) && $passedAttributes['data-title-override'] === 'true')) ? $passedAttributes['title'] : $systemImage->getProperty('title');
+                    $alt = ($passedAttributes['alt'] || (array_key_exists('alt', $passedAttributes) && $passedAttributes['data-alt-override'] === 'true')) ? $passedAttributes['alt'] : $systemImage->getProperty('alternative');
```

Comment:
See my comment in conversation: this is too complex to be written down in one line of code, should be moved into own function.